### PR TITLE
feat: Added yaml ext to base container.

### PR DESCRIPTION
**Pull request by andre-at-situ**

### CODE REVIEW

**Review by Sassiest01 — APPROVED** [review]

File: templates/python/.devcontainer.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
     "customizations": {
         "vscode": {
             "extensions": [
+                "be5invis.toml",
```
Comment (willsawyerrrr) [inline templates/python/.devcontainer.json:15]: What wasn't working about the other one? My experience with Even Better TOML has been great.

**Comment by willsawyerrrr** [comment]
What wasn't working with the Even Better TOML extension? My experience with it has been great.

**Comment by andre-at-situ** [comment]
> What wasn't working with the Even Better TOML extension? My experience with it has been great.

It doesn't highlight pyproject.toml

**Comment by willsawyerrrr** [comment]
> > What wasn't working with the Even Better TOML extension? My experience with it has been great.
> 
> It doesn't highlight pyproject.toml

It works for me. Is it definitely installed correctly in your container?

image

**Comment by andre-at-situ** [comment]
> > > What wasn't working with the Even Better TOML extension? My experience with it has been great.
> > 
> > 
> > It doesn't highlight pyproject.toml
> 
> It works for me. Is it definitely installed correctly in your container?
> 
> image

ofc .. it's always flaky af for me.. in any case I've reverted it .. I'll just replace locally.

**Review by willsawyerrrr — APPROVED** [review]

**Pull request merged**